.ado/templates/react-native-init.yml

```diff
@@ -88,10 +88,16 @@ steps:
       msbuildArguments:
         /p:PreferredToolArchitecture=$(MSBuildPreferredToolArchitecture)
         /p:PlatformToolset=$(MSBuildPlatformToolset)
+        /bl
 #        /p:VCTargetsPath="C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\Microsoft\VC\v150"
 
       clean: true # Optional
 
+  - task: PublishBuildArtifacts@1
+    inputs:
+      displayName: 'Publish testcli binlog'
+      pathToPublish: $(Agent.BuildDirectory)\testcli\windows\msbuild.binlog
+
```

Comment:
I'm still experimenting with this to get it right.  Once it works, I'll likely add this to the other things we're building in the CI loop.